support-cycle.md

```diff
@@ -0,0 +1,85 @@
+
+##### Validating Node.js Support
+
+We validate our support for Node.js by running Continuous Integration testing our projects in the latest release of every major version - for example at the time of writing the current LTS release version is `6.11.13`; we will test this version but none of the earlier releases in the 6.0 branch.
+
+#### Firefox
+
+We support Firefox from it's _Extended Support Release_ (ESR) up until the _latest_ released version. [Firefox's ESR release cycle is documented here](https://www.mozilla.org/en-US/firefox/organizations/faq/) and can be effectively summarised as: a new ESR release is made approximately every year, while new regular releases are made every 6 weeks. For us this means we essentially support a years worth of Firefox releases.
+
+##### Validating Firefox Support
+
+We validate our support for Node.js by running Continuous Integration testing our projects in both the latest ESR of Firefox, as well as the latest version of Firefox. Any versions inbetween are not tested as it is unlikely that there will be failures in specific versions between the ESR and latest releases that aren't reflected in one or the other.
+
+#### Chrome
+
+Chrome has no Extended or Long Term Support cycle, and as such only the latest version so Chrome is officially supported by Google. We provide soft-support for Chrome releases up to a year old (that is, while we do not test any older releases, if a user finds a bug we will fix it).
+
+##### Validating Chrome support
+
+We validate our support for Chrome by running Continuous Integration testing for the latest version of Chrome. Older versions aren't tested because this is not easily available in our Continuous Integration environments.
+
+#### Safari (MacOS and iOS)
+
+Safari is tied to the MacOS and iOS operating system releases, of which Apple supports the last 3 releases in each (this equates to roughly 3 years, as a new MacOS and iOS release is made every year), however Apple also releases updates for Safari versions for the last 3 _desktop_ operating system versions. [Apple's support notes for security releases can be found here](https://support.apple.com/en-us/HT201222). Information around Safari releases can be a little tricky to find, and Apple does not have sufficient support documentation around this - the best place to find this information is on the [Wikipedia entry for Safari version history](https://en.wikipedia.org/wiki/Safari_version_history). To demonstrate how the release cycle works; iOS 8 and MacOS 10.10 both shipped with Safari 8, iOS9 and MacOS 10.11 shipped with Safari 9, iOS10 and MacOS 10.12 shipped Safari 10 and iOS11 and MacOS 10.13 shipped with Safari 11. Safari 10 was backported as an update to MacOS 10.11 and 10.10 users, as was Safari 11 to MacOS 10.12 and 10.11 users. The iOS version of Safari remains fixed.
```

Comment:
I think that this fails to say which versions will be supported by chai
I can infer that it is the last 3 releases, but it is not explicitly said as in Firefox for example:
`For us this means we essentially support a years worth of Firefox releases.`